src/core/playground/scheme.ts

```diff
@@ -0,0 +1,4 @@
+export enum SignatureScheme {
```

Comment:
I think we already have equivalent -> SigningScheme.

Reply:
note: there is SigningScheme and SigningSchemeInput.
SigningScheme represents all SigningSchemes (ed,multied,single signer, multi signer)
SigningSchemeInput represents what we accept (ed,secp)